__tests__/main.test.js

```diff
@@ -3,6 +3,7 @@ const app = require('../app')
 const seed = require('../db/seeds/seed')
 const data = require('../db/data/test-data/index')
 const db = require('../db/connection')
+require('jest-sorted')
```

Comment:
There's no need to require jest-sorted in if it has been installed

Reply:
I think in my test it is not working without requiring the package. not recognising the function

i have installed it with --save -d